public/src/reducers/script.jsx

```diff
@@ -0,0 +1,38 @@
+const scriptInfo = {
+  name: null,
+  body: null,
+  description: null
+};
+
+const scriptQuestions = [];
+
+export const SET_SCRIPT_INFO = 'SET_SCRIPT_INFO';
+export const SET_SCRIPT_QUESTIONS = 'SET_SCRIPT_QUESTIONS';
+
+export function scriptInfoReducer(state = scriptInfo, action) {
```

Comment:
After we combine the actions and sort out integration with the script list view I don't think this will be necessary. Looks good though.